sdks/python/apache_beam/io/gcp/internal/clients/storage/storage_v1_client.py

```diff
@@ -52,7 +52,7 @@ def __init__(self, url='', credentials=None,
     super(StorageV1, self).__init__(
         url, credentials=credentials,
         get_credentials=get_credentials, http=http, model=model,
-        log_request=log_request, log_response=log_response,
+        log_request=log_request, log_response=log_response, num_retries=20,
```

Comment:
This client is autogenerated so this code might get replaced when this client is generated next time. Can you also  send a PR to apitools so that the default number of retries is increased for the GCS client.